src/Swarm/Language/LSP/Hover.hs

```diff
@@ -36,9 +56,12 @@ showHoverInfo _ _ p vf@(VirtualFile _ _ myRope) =
   case readTerm' content of
     Left _ -> Nothing
     Right Nothing -> Nothing
-    Right (Just stx) -> Just (treeToMarkdown 0 $ explain term, finalPos)
+    Right (Just stx@(Syntax l t)) -> Just (treeToMarkdown 0 $ explain Long mMod term, finalPos)
      where
-      Syntax sloc term = narrowToPosition stx $ fromIntegral absolutePos
+      (betterTerm, mMod) = case processParsedTerm stx of
+        Left _e -> (t, Nothing)
+        Right (ProcessedTerm betterT modul _req _reqCtx) -> (betterT, Just modul)
```

Comment:
See my comment below re: `force`.  I think we *don't* want to use `betterT` here, but always explain things in terms of the user's original parsed syntax `stx`.  Of course we still do want to use the output of typechecking (here, `modul`).

Reply:
oh, now I scrolled to this comment. 😄 Will do.